jobs/uaa-customized/templates/web/external_auth_error.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org" xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout" layout:decorator="layouts/main">
+<head>
+</head>
+<body>
+<div class="govuk-grid-row" layout:fragment="page-content">
+  <div class="govuk-grid-column-two-thirds">
+    <div th:if="${param.error}" class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="error-summary">
+      <h2 class="govuk-error-summary__title" id="error-summary-title">
+          There was an error when authenticating against the external identity provider. You may need to:
+      </h2>
+      <div class="govuk-error-summary__body" th:if="${#strings.indexOf(param.error[0], 'The user account must be pre-created')}">
```

Comment:
If the error message does not contain this string, it seems like we do not display a message to the user?